benchmarkController/hades-controller.go

```diff
@@ -1,21 +1,33 @@
 package benchmarkController
 
 import (
-	"log/slog"
-
 	"github.com/Mtze/CI-Benchmarker/executor"
 	"github.com/Mtze/CI-Benchmarker/persister"
+	"github.com/joho/godotenv"
+	"log/slog"
+	"os"
 )
 
-const (
-	//TODO: read this from config
-	hades_host = "http://localhost:8081/build"
-)
+var hadesHost string
+
+func init() {
+	// Load .env file
+	err := godotenv.Load()
```

Comment:
I think it would be a good idea to use [viper](https://github.com/spf13/viper) from now on. Can you please add this here?